file-server/src/test/java/se/fortnox/reactivewizard/fileserver/FileServerTest.java

```diff
@@ -0,0 +1,35 @@
+package se.fortnox.reactivewizard.fileserver;
+
+import io.reactivex.netty.protocol.http.server.MockHttpServerRequest;
+import org.junit.Test;
+import rx.Observable;
+import se.fortnox.reactivewizard.mocks.MockHttpServerResponse;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+public class FileServerTest {
```

Comment:
Maybe add some more unit test for the requesthandler when the path is not set in the config, as well when the index file is specified in the config and that utf-8 is specified when js files. Just to not reduce code coverage in project in overall. 

as well, the test class should have the same name as the class it tests